src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,34 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = super.factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't save actor to DB", e);
```

Comment:
Make all messages informative
```suggestion
            throw new DataProcessingException("Can't save actor to DB " + actor , e);
```